### DISCUSSION TITLE
Has Prometheus been cracked/deobsfucated

### DISCUSSION THREAD

**Discussion by Brohammer5** [prometheus-lua/Prometheus#28] [Q&A] [answered]
Can Prometheus been cracked/deobsfucated?

**Comment by levno-710** [comment, marked as answer]
As far as I know, there is no automated deobfuscation tool, however the VM is currently extremely slow and I am in the moment working on a better vm to increase speed.
Another problem that the new vm in the next release will fix, is that it is relatively easy to dump constants used by the old vm.

**Reply by levno-710** [reply]
Update: Vm is no longer slow
